acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/pages/DrivingLicenceAPIPage.java

```diff
@@ -171,6 +173,72 @@ public void postRequestToDrivingLicenceEndpoint(String drivingPermitJsonRequestB
         postRequestToDrivingLicenceEndpoint(drivingPermitJsonRequestBody, "");
     }
 
+    public void postRequestToDrivingLicenceEndpointWithInvalidSessionId(
+            String invalidHeaderValue, String dlJsonRequestBody, String jsonEditsString)
+            throws IOException, InterruptedException, NoSuchFieldException, IllegalAccessException {
+        String privateApiGatewayUrl = configurationService.getPrivateAPIEndpoint();
+        JsonNode dlJsonNode =
+                objectMapper.readTree(
+                        new File("src/test/resources/Data/" + dlJsonRequestBody + ".json"));
+        String dlInputJsonString = dlJsonNode.toString();
+
+        Map<String, String> jsonEdits = new HashMap<>();
+        if (!StringUtils.isEmpty(jsonEditsString)) {
+            jsonEdits = objectMapper.readValue(jsonEditsString, Map.class);
+        }
+
+        DrivingPermitForm drivingPermitFormJson =
+                objectMapper.readValue(
+                        new File("src/test/resources/Data/" + dlJsonRequestBody + ".json"),
+                        DrivingPermitForm.class);
+
+        for (Map.Entry<String, String> entry : jsonEdits.entrySet()) {
+            Field field = drivingPermitFormJson.getClass().getDeclaredField(entry.getKey());
+            field.setAccessible(true);
+
+            field.set(drivingPermitFormJson, entry.getValue());
+        }
+        String drivingPermitInputJsonString =
+                objectMapper.writeValueAsString(drivingPermitFormJson);
+
+        HttpRequest.Builder builder =
+                HttpRequest.newBuilder()
+                        .uri(URI.create(privateApiGatewayUrl + "/check-driving-licence"))
+                        .setHeader("Accept", "application/json")
+                        .setHeader("Content-Type", "application/json")
+                        .POST(HttpRequest.BodyPublishers.ofString(drivingPermitInputJsonString));
+
+        switch (invalidHeaderValue) {
+            case "invalidSessionId" -> builder.setHeader(
+                    "session_id", UUID.randomUUID().toString());
```

Comment:
Is this not setting a valid instead of invalid sessionId by setting the value to a UUID?

Reply:
Thats correct, it sets the session_id to a random UUID (invalid) instead of a Valid UUID generated from the /session endpoint

Reply:
Ah, so this scenario covers if a valid sessionId doesn't match the users sessionID? In which case for clarity can we call the case "sessionIdMismatch" or something along those lines.

Reply:
Scenario example has been updated to mismatchSessionIdm thanks